src/main/java/hudson/plugins/git/browser/AssemblaWeb.java

```diff
@@ -83,12 +84,14 @@ public URL getFileLink(Path path) throws IOException {
 
     @Extension
     public static class AssemblaWebDescriptor extends Descriptor<RepositoryBrowser<?>> {
+        @Nonnull
         public String getDisplayName() {
             return "AssemblaWeb";
         }
 
         @Override
-        public AssemblaWeb newInstance(StaplerRequest req, JSONObject jsonObject) throws FormException {
+        public AssemblaWeb newInstance(StaplerRequest req, @Nonnull JSONObject jsonObject) throws FormException {
```

Comment:
these seem like unrelated changes... ideally the core version bump would be in a separate PR... not saying to move it out now... just noting for future code archeologists

Reply:
Yepp, I kept all those changes in it's own commit so that it can be reverted if it was too much in the PR, but I can't get a green build without it. I consulted with @MarkEWaite on what to do about it :)

The core and plugin bumps is needed for the built storm fix and something else I can't remember at the moment.